### ISSUE TITLE
Should the extension allow serving multiple project files concurrently, from the same workspace?

### ISSUE THREAD

**Issue by evaera** [rojo-rbx/vscode-rojo#44]
Right now, multiple project files can be serving concurrently, but only if they are from different workspaces (this happens have a multi-root workspace). If you have a project serving already, and you try to start a project file from the same workspace, the old one is stopped before starting the new one. 

Is this the behavior that most people want? Or do most people want to be able to serve multiple project files from the same project at once?

**Comment by testpossessed** [comment]
Definitely a nice to have.  For example I am working on a game that has mulitple levels.  The starter place is where the player starts and can select a mission or view the global leader board.  All are in one repo and I use folders for each place in a single workspace.  Sometimes it would be nice to be able to have both the starter place and the mission place I am working on open in studio and syncing to separate Rojo projects.  I have forgotten to switch projects a couple of times and ended up connecting to the wrong project.

However not sure it should be a high priority feature.
